packages/shared/world/types.ts

```diff
@@ -0,0 +1,21 @@
+import { ILand } from 'shared/types'
+
+export enum IsolatedMode {
+  BUILDER = 0 // Payload to use: BuilderIsolatedModeOptions
+}
+
+export type IsolatedModeOptions = {
```

Comment:
To avoid doing `as SomeType` when we want to cast the payload.
```suggestion
export type IsolatedModeOptions<T = any> = {
  mode: IsolatedMode
  payload: T
}
```